docs/content/Select.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Select
```

Comment:
I'm thinking maybe "SelectInput" is a better name. I'd love to hear what other reviewers think.

Reply:
Yep! I think so too. It kinda cements it as a Form component and not any fancy menu/overlay interface

Reply:
We only recently decided to avoid using `*Input` suffix on form components, so I'm in favour of keeping it as `Select` here. Reason: a) To match the underlying HTML and b) be consistent with other form components like Radio, Checkbox.